packages/gatsby-source-strapi/.npmignore

```diff
@@ -0,0 +1,35 @@
+# Logs
+logs
```

Comment:
I don't think any of this file is needed, looks like other packages vary. Lmk if you have thoughts.